doc/releasenotes/15_0_0_summary.md

```diff
@@ -248,3 +248,34 @@ send semi-sync ACKs. This ensures that any committed write exists in at least 2
 #### FORMAT=vtexplain
 
 With this new `explain` format, you can get an output that is very similar to the command line `vtexplain` app, but from a running `vtgate`, through a MySQL query.
+
+### VTOrc
+
+#### Configuration
+
+VTOrc configurations that had `Orchestrator` as a substring have been renamed to have `VTOrc` instead. The old configuration won't 
+work in this release. So if backward compatibility is desired, then before upgrading it is suggested to copy the old configurations
```

Comment:
I am little confuse here.. for backward compatibility should we do as suggested 

<quotes>So if backward compatibility is desired, then before upgrading it is suggested to copy the old configurations</quotes>

1- first we copy old config to new configurations
2- We then roll new bits
3- remove the old config from configurations

OR

<quotes>VTOrc ignores the configurations that it doesn't understand. So new configurations can be added while running the previous release.</unquotes>

1- first we copy new config to existing configurations
2- We then roll out new bits
3- remove the old config from configurations

Reply:
I have improved the summary doc and added an example too to make it clear.